app.js

```diff
@@ -1,3 +1,5 @@
+dotenv = require('dotenv');
```

Comment:
I'd love a simple one-liner `require('dotenv').config()`

Reply:
you got it